src/shaders/_prelude_shadow.fragment.glsl

```diff
@@ -91,6 +91,30 @@ highp float shadow_occlusion_0(highp vec4 pos, highp float bias) {
     return clamp(value / 9.0, 0.0, 1.0);
 }
 
+float shadow_occlusion(highp vec4 light_view_pos0, highp vec4 light_view_pos1, float view_depth, highp float bias) {
+    const float cascadeFadeRange = 0.05;
+    const float endFadeRange = 0.25;
```

Comment:
Maybe those two constants could be configurable via uniform in the future. Not needed for now and out of the scope of this PR.